File: gr00t/Isaac-GR00T/examples/H20/robots/sdk_package/example/h2x/system/clear_error.cpp
// clear_error.cpp

#include <iostream>
#include <iomanip>
#include "h2x/h2x_system.h"
#include "common/common_def.h"

using namespace humanoid;
using namespace humanoid::h2x;

int main() {
  std::cout << "=== 清除所有报错信息参数示例 ===" << std::endl;
  H2xSystem system;
  std::cout << "清除报错信息" << (system.clearError() == 0 ? "成功" : "失败")
            << std::endl;
  return 0;
}

File: gr00t/Isaac-GR00T/examples/H20/robots/sdk_package/example_py/h2x/system/clear_error.py
import humanoid_sdk_py
import humanoid_sdk_py.h2x as h2x
import time

h2x_system = h2x.H2xSystem()
time.sleep(1)

print("=== 清除所有报错信息参数示例 ===")
ret = h2x_system.clearError()
if ret == 0:
    print("清除报错信息成功")
else:
    print("清除报错信息失败")
